tests/gui_tests/interaction/CMakeLists.txt

```diff
@@ -4,33 +4,17 @@ project(InteractionTests LANGUAGES CXX)
 
 enable_testing()
 
-find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Gui Quick Test)
-find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Gui Qml Quick Test Concurrent)
-find_package(Qt6 REQUIRED COMPONENTS QuickControls2)
-
-# add_subdirectory(suri)
+find_package(Qt6 REQUIRED COMPONENTS Quick Qml QuickControls2 Test Concurrent)
 
 set(CMAKE_AUTOUIC ON)
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
We already have these two lines in CMakeLists.txt for gui_tests. Is there a need to have them also in gui_tests_interactions/CMakeLists.txt?